### PR TITLE
Sharing info with cloud apps

### PR DESCRIPTION
**Pull request by tpmccallum**
Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [x] The `title`, `template`, and `date` are all set
- [x] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [x] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep $'\r' | wc -l` and expect 0 as a result)
- [x] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)
- [x] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)
- [x] Headings are using Title Case
- [x] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [x] Have tested with `npm run test` and resolved all errors
- [x] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

### CODE REVIEW

File: content/cloud/linking-applications-to-resources-using-labels.md [hunk 1]
```diff
@@ -9,9 +9,11 @@ url = "https://github.com/fermyon/developer/blob/main/content/cloud/linking-appl
...
+Many Spin applications require data storage between invocations. Fermyon Cloud offers two methods for this: Key Value storage and SQLite databases. However, the relationship between applications and databases isn't strictly one-to-one. A single database might be used by multiple applications, while an application could utilize several databases. There might even be a need to maintain a database that isn't utilized by any applications, perhaps for analytics or compliance reasons. All of these scenarios can occur as applications are independently deployed, upgraded, and retired, without any particular application needing to be aware of how others might be using the same databases.
```
Comment (itowlson) [inline content/cloud/linking-applications-to-resources-using-labels.md:16]: Have we capitalised this elsewhere?  I haven't been doing so, but I see you've updated this in a few places - is this "official" style now and should we be porting it across the docs and the UI?

File: content/cloud/linking-applications-to-resources-using-labels.md [hunk 2]
```diff
@@ -144,6 +146,52 @@ $ spin cloud kv list
 
 Now using these commands you can see your Spin applications, their respective labels and their connected databases and stores.
 
+### Linking Resources Between Applications
+
+Linking SQLite and Key Value resources between applications in Fermyon Cloud is possible. Here is an example of how you would link (share) a Key Value resource between two applications in Fermyon Cloud. Let's start by creating a new Key Value resource:
```
Comment (itowlson) [inline content/cloud/linking-applications-to-resources-using-labels.md:151]: "is possible" read a bit oddly to me as a sentence ending.  How about e.g.

```suggestion
Sometimes, you want applications to share persistent data stores.  For example, you might have a shopping application and its admin console deployed as separate applications, but sharing data.  You can do this in Fermyon Cloud by linking the same SQLite and Key Value resources to multiple applications. Here is an example of how you would link (share) a Key Value resource between two applications in Fermyon Cloud. Let's start by creating a new Key Value resource:
```

File: content/cloud/linking-applications-to-resources-using-labels.md [hunk 3]
```diff
@@ -144,6 +146,52 @@ $ spin cloud kv list
...
+
+<!-- @selectiveCpy -->
+
+```bash
+$ spin cloud key-value create my-shared-kv
+Key value store  "my-shared-kv" created
```
Comment (itowlson) [inline content/cloud/linking-applications-to-resources-using-labels.md:157]: Is the extra space after `store` there in the console output?  If so we should remove that...

**Review by itowlson — APPROVED** [review]
I left a wording suggestion but not a blocker - LGTM

File: content/cloud/linking-applications-to-resources-using-labels.md (hunk 1 above)
Comment (tpmccallum) [reply content/cloud/linking-applications-to-resources-using-labels.md:16]: Not 100% sure TBH but I did a search for `Key Value` vs `key-value` and went with the majority wins. Happy to use Key Value moving forward, yeah.

File: content/cloud/linking-applications-to-resources-using-labels.md (hunk 3 above)
Comment (tpmccallum) [reply content/cloud/linking-applications-to-resources-using-labels.md:157]: Yes, the extra space is present on the `create` subcommand. 
```
Key value store  "my-shared-kv-test" created
```

Comment (tpmccallum) [reply content/cloud/linking-applications-to-resources-using-labels.md:157]: https://github.com/fermyon/cloud-plugin/pull/197/files

**Comment by tpmccallum** [comment]
Thanks @itowlson

File: content/cloud/linking-applications-to-resources-using-labels.md (hunk 1 above)
Comment (itowlson) [reply content/cloud/linking-applications-to-resources-using-labels.md:16]: It was more the capitalisation - `key value` vs `Key Value`.  Agree we are inconsistent about the hyphenation

**Pull request merged**